_includes/templates/corp-info-inst.html

```diff
@@ -0,0 +1,71 @@
+{%- comment -%}
```

Comment:
How different this one is compared to "_includes/templates/corp-info-org.html" from a component perspective?

Like identifying what is different and can be adapated.

Reply:
> How different this one is compared to "_includes/templates/corp-info-org.html" from a component perspective?
> 
> Like identifying what is different and can be adapated.

The content in the "col-md-8" is different. There is probably a way to merge them and have parameters in the include, but it should be handled in a different PR.